### ISSUE TITLE
Update to minecraft 1.16.2

### ISSUE THREAD

**Issue by LukasmanMHdude** [Minehut/Meta#256]
### Feature request

**Platform**
website, ingame

**Describe the feature**
![image](https://user-images.githubusercontent.com/69743171/92649518-01f8e780-f2a0-11ea-8e3f-e046e9d16aff.png)
![image](https://user-images.githubusercontent.com/69743171/92649524-058c6e80-f2a0-11ea-8894-f6d0aef5bdd4.png)
![image](https://user-images.githubusercontent.com/69743171/92649556-11783080-f2a0-11ea-9296-3a44343c2ebd.png)
![image](https://user-images.githubusercontent.com/69743171/92649590-1e951f80-f2a0-11ea-9457-9178328ba866.png)
![image](https://user-images.githubusercontent.com/69743171/92649630-2e146880-f2a0-11ea-92f8-d74cb97a6fbd.png)

Update to 1.16. Add a button to not receive plugin updates/version updates from my that lasts a week or 2. (for compatibility so we don't have the same issue as 1.16 update)

**How the feature is useful**
features
![Annotation 2020-07-11 210357](https://user-images.githubusercontent.com/69743171/92665883-8bb6ae00-f2bc-11ea-8730-409a97a3d8ec.png)

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by WallyTube** [comment]
It is a no-brainer that minehut will update to 1.16.2 as soon as possible. They are probably waiting on plugin or paper dependencies to roll out stable builds for 1.16.2 In the meantime, you still can use 1.16.2 on 1.16.1 servers, and I'm not really sure what is different between the 2 versions except for bug updates.

**Comment by Shlba** [comment]
> ### Feature request
> **Platform**
> website, ingame
> 
> **Describe the feature**
> ![image](https://user-images.githubusercontent.com/69743171/92649518-01f8e780-f2a0-11ea-8e3f-e046e9d16aff.png)
> ![image](https://user-images.githubusercontent.com/69743171/92649524-058c6e80-f2a0-11ea-8894-f6d0aef5bdd4.png)
> ![image](https://user-images.githubusercontent.com/69743171/92649556-11783080-f2a0-11ea-9296-3a44343c2ebd.png)
> ![image](https://user-images.githubusercontent.com/69743171/92649590-1e951f80-f2a0-11ea-9457-9178328ba866.png)
> ![image](https://user-images.githubusercontent.com/69743171/92649630-2e146880-f2a0-11ea-92f8-d74cb97a6fbd.png)
> 
> Update to 1.16. Add a button to not receive plugin updates/version updates from my that lasts a week or 2. (for compatibility so we don't have the same issue as 1.16 update)
> 
> **How the feature is useful**
> features
> ![Annotation 2020-07-11 210357](https://user-images.githubusercontent.com/69743171/92665883-8bb6ae00-f2bc-11ea-8730-409a97a3d8ec.png)
> 
> **Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

maybe follow the format like jellz stated in another post?

**Comment by LukasmanMHdude** [comment]
wdym?

**Comment by WallyTube** [comment]
was quoting the entire thread necessary?

**Comment by MiginsMC** [comment]
Paper has already released a stable build of 1.16.2 I believe. 
But please don't rush this update like the 1.16 one, take precautions, take your time. We don't want worlds resetting again :eyes:

**Comment by LukasmanMHdude** [comment]
Thats why theres a button to stay behind

**Comment by MiginsMC** [comment]
> Thats why theres a button to stay behind, that will be taken away once everything is 100% good

I don't like the idea of this 'button'. That seems like an unnecessarily large amount of effort for Minehut to put in for a single release.

**Comment by LukasmanMHdude** [comment]
People have been requesting a toggle like this for a while, and once its implemented once it can be used for future Minecraft releases

**Comment by LukasmanMHdude** [comment]
(to prevent the complaining that came with the 1.16 update)

**Comment by MiginsMC** [comment]
1. You can edit your messages here so there's no need to send two.
2. If you're suggesting that be made, you should put that in another issue, because people may have different opinions on your two suggestions. I don't think it's necessary at all and Minehut should just take their time in releasing.

**Comment by LukasmanMHdude** [comment]
Thanks for the info mig!

**Comment by drtshock** [comment]
We're on 1.16.3 now.
